frontend/views/components/CircularSlider/components/CircleSlider.vue

```diff
@@ -1,9 +1,8 @@
-<template lang="pug">
+<template lang='pug'>
   div
     svg(
-      :width="side + 'px'"
-      :height="side + 'px'"
-      :viewbox="'0 0 ' + side + ' ' + side"
+      :width='side + "px"'
+      :viewbox='"0 0 " + side + " " + side'
```

Comment:
Are we missing the `:height` now? Was removing that intentional?

Reply:
I'm already sleeping. Now it should be 💯

Reply:
You sure? Now it's reverted to the old order of the quotes, which this PR is trying to fix 😁

I recommend getting some rest and when you return, before pushing any other fixes, do as I'm doing and go through this PR yourself line-by-line, double-checking everything like a compiler. :)